File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/DescripcionesPreferidasPorRefSetResponse.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for descripcionesPreferidasPorRefSetResponse complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="descripcionesPreferidasPorRefSetResponse">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element ref="{http://service.ws.semantikos.minsal.cl/}respuestaConceptosPorRefSet" minOccurs="0"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "descripcionesPreferidasPorRefSetResponse", propOrder = {
    "respuestaConceptosPorRefSet"
})
public class DescripcionesPreferidasPorRefSetResponse {

    @XmlElement(namespace = "http://service.ws.semantikos.minsal.cl/")
    protected RespuestaConceptosPorRefSet respuestaConceptosPorRefSet;

    /**
     * Gets the value of the respuestaConceptosPorRefSet property.
     * 
     * @return
     *     possible object is
     *     {@link RespuestaConceptosPorRefSet }
     *     
     */
    public RespuestaConceptosPorRefSet getRespuestaConceptosPorRefSet() {
        return respuestaConceptosPorRefSet;
    }

    /**
     * Sets the value of the respuestaConceptosPorRefSet property.
     * 
     * @param value
     *     allowed object is
     *     {@link RespuestaConceptosPorRefSet }
     *     
     */
    public void setRespuestaConceptosPorRefSet(RespuestaConceptosPorRefSet value) {
        this.respuestaConceptosPorRefSet = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/descripcionesPreferidasPorRefSetResponse.php
<?php

namespace Semantikos\ClientBundle\API;

class descripcionesPreferidasPorRefSetResponse
{

    /**
     * @var RespuestaConceptosPorRefSet $respuestaConceptosPorRefSet
     */
    protected $respuestaConceptosPorRefSet = null;

    /**
     * @param RespuestaConceptosPorRefSet $respuestaConceptosPorRefSet
     */
    public function __construct($respuestaConceptosPorRefSet)
    {
      $this->respuestaConceptosPorRefSet = $respuestaConceptosPorRefSet;
    }

    /**
     * @return RespuestaConceptosPorRefSet
     */
    public function getRespuestaConceptosPorRefSet()
    {
      return $this->respuestaConceptosPorRefSet;
    }

    /**
     * @param RespuestaConceptosPorRefSet $respuestaConceptosPorRefSet
     * @return descripcionesPreferidasPorRefSetResponse
     */
    public function setRespuestaConceptosPorRefSet($respuestaConceptosPorRefSet)
    {
      $this->respuestaConceptosPorRefSet = $respuestaConceptosPorRefSet;
      return $this;
    }

}
